tests/Certificationy/QuestionTest.php

```diff
@@ -100,16 +99,16 @@ public function testAreCorrectAnswers()
      */
     public function testIsMultipleChoice()
     {
-        $multipleChoiceAnswers = array(
+        $multipleChoiceAnswers = new Answers([
```

Comment:
was bad idea, we don't want to create hard bound between `Answers` utility class and the implementation, the Collection should only depends on the interface.

Moved to `TestCase` class instead.